lib/omniauth/strategies/younility.rb

```diff
@@ -8,8 +8,8 @@ class Younility < OmniAuth::Strategies::OAuth2
 
       option :client_options, {
         site:           'https://app.younility.com',
```

Comment:
I'm able to change this in the web app. Right now there's an environment variable that the web app looks for. If it's there it uses the variable, if not it will just use the default one.